draft-ietf-rtcweb-jsep.xml

```diff
@@ -2146,12 +2146,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             lines are not meaningful in the JSEP context and
             MAY be discarded once they have been checked.
             <list>
+              <t>The "c=" line MUST be checked for syntax but its
+              value is not used.  This supersedes the guidance in <xref
+              target="RFC5245"/>, Section 6.1, to use "ice-mismatch" to indicate
+              mismatches between "c=" and the candidate lines. Because JSEP always
```

Comment:
would sound better as 
"... and the candidate lines; because JSEP always uses ICE, "ice-mismatch" is not useful in this context."